web/wp-content/themes/dctx-v2/header.php

```diff
@@ -0,0 +1,72 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-fciv2p.php, but found phpcs-scan-FCiv2P.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).